model/riscv_crypto_entropy_source.sail

```diff
@@ -0,0 +1,99 @@
+/*
+ * Contains a basic model for the RISC-V cryptography extension entropy
+ * source. See section 4 of the specifications for details.
+ * - https://github.com/riscv/riscv-crypto/releases
+ */
+
+/*
+ * The mnoise CSR
+ * ------------------------------------------------------------
+ */
+
+/* Bitfield definition for the mnoise CSR.
+ * Most of this is implementation defined, so only the architectural bit
+ * `NOISE_TEST` is actually specified here.
+ */
+bitfield Mnoise : xlenbits = {
+  NOISE_TEST : 31
+}
+
+register mnoise : Mnoise
+
+/* mnoise is defined in machine mode only */
+function clause ext_is_CSR_defined (0x7A9, Machine) = {
```

Comment:
```suggestion
function clause ext_is_CSR_defined(0x7A9, Machine) = {
```
is the right style. Also repeated below for mentropy and some of your other functions. The exception in the style seems to be `execute`, for whatever reason that one consistently _does_ have the space (though there are a fair number of cases for other functions too... 😞).

Reply:
Okay, I think I've caught all the non-execute clause cases of this. I've left execute clauses with a space.